health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/ViewHierarchy.js

```
@@ -151,47 +150,218 @@ const ViewHierarchy = () => {
+          });
+      
+          const id = createResponse?.ResourceDetails?.id;
+          const typeOfData = createResponse?.ResourceDetails?.type;
+      
+          if (id) {
+            try {
+              await pollForStatusCompletion(id, typeOfData);
+              setDataCreateToast(false);
+              setShowToast({ label: `${t("WBH_HIERARCHY_CREATED")}`, isError: "success" });
+            } catch (pollError) {
+              throw pollError; // Propagate polling errors to the outer catch block
```

Biome

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)